### ISSUE TITLE
Compatibility with Django compressor?

### ISSUE THREAD

**Issue by itbabu** [divio/django-cms#5454]
I'm using django-compressor with sekizai:
```Django
{% render_block "js" postprocessor "compressor.contrib.sekizai.compress" %}
```
After this commit https://github.com/divio/django-cms/commit/4f2ec2361b0fef3c7e84d553ca86819c23debd43 a script with type ```text/cms-template```  was moved  inside an addtoblock "js" block. 

Django-compressor doesn't know how to handle it and it raises an error.
 So when I try to edit a page I get:
```
Couldn't find any precompiler in COMPRESS_PRECOMPILERS setting for mimetype 'text/cms-template'.
```
Here the file: [cms/templates/cms/toolbar/placeholder.html](https://github.com/divio/django-cms/blob/release/3.3.x/cms/templates/cms/toolbar/placeholder.html)

Is it possible to move the script outside the addtoblock "js" block and the language block?

**Comment by czpython** [comment]
@itbabu 
Thanks for the report.
@vxsx let's discuss.

**Comment by vxsx** [comment]
@czpython separate sekizai block?

**Comment by vxsx** [comment]
Should be fixed by #5463

**Comment by czpython** [comment]
Fixed by https://github.com/divio/django-cms/pull/5463
